internal/locald/rootmanager/root_manager.go

```diff
@@ -132,9 +132,9 @@ func (m *rootManager) runSandboxManager(ctx context.Context) (err error) {
 			cmdToRun.Stdout = os.Stdout
 			cmdToRun.Stdin = os.Stdin
 			// Prevent signaling the children
-			//cmdToRun.SysProcAttr = &syscall.SysProcAttr{
-			//	Setpgid: true,
-			//}
+			cmdToRun.SysProcAttr = &syscall.SysProcAttr{
+				Setpgid: true,
+			}
```

Comment:
Without this, after `ctrl-c` the sandbox-manager was still running:

```bash
ddv@ddvpc:~/git/signadot/cli$ ps ax | grep "signadot local"
 212098 pts/0    Sl+    0:00 ./signadot local connect
 212110 ?        Ss     0:00 sudo -S --preserve-env=SIGNADOT_LOCAL_CONNECT_INVOCATION_CONFIG ./signadot locald
 212127 ?        Sl     0:00 ./signadot locald
 212140 ?        S      0:00 sudo -n -u #1000 --preserve-env=SIGNADOT_LOCAL_CONNECT_INVOCATION_CONFIG ./signadot locald
 212141 ?        Sl     0:00 ./signadot locald

ddv@ddvpc:~/git/signadot/cli$ ps ax | grep "signadot local"
 212141 ?        Sl     0:00 ./signadot locald
```

Reply:
I am unable to replicate this (on Mac). Also, it makes sense to me that the sandbox manager should be in the same process group as its caller, so that signals get sent to it when sent to its caller.  I guess on a Mac (or likely any BSD based system), when the parent exits it kills its children and on linux, not (at least under the Go libs we're using). 

However, since the `signadot local` invocation is the only thing getting a signal, I'd expect the root manager should shutdown its children regardless.  I'd suggest the root manager explicitly kill the sandbox manager on shutdown rather than relying on subtle platform dependent IPC.  I know we needed that for the root sudo but otherwise I think it's undesirable.

Reply:
following discussion, it seems like this is still about sudo (we had different versions) rather than MacOS.  Please consider resolved.